addons/debugger/addon.json

```diff
@@ -73,24 +73,21 @@
       "id": "log_invalid_cloud_data",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Fill and animate graphs",
+      "id": "fancy_graphs",
+      "type": "boolean",
+      "default": true
```

Comment:
I suggest we change the default

Reply:
So turning off the fill and animation by default?

Reply:
Yeah, I guess it makes sense to have the best setting for performance by default.

Reply:
Done. Do you think this needs migration code? probably not.

Reply:
This setting wasn't available previously so there's nothing to migrate.
We're assuming existing users of this addon will not be affected by this change. They can always change it back.